src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -210,7 +210,7 @@ def get_pfs_version():
 def is_pfs_service_healthy(pfs_port) -> bool:
     """Check if pfs service is running and pfs version matches pf version."""
     try:
-        response = requests.get("http://localhost:{}/heartbeat".format(pfs_port))
+        response = requests.get("http://127.0.0.1:{}/heartbeat".format(pfs_port))
```

Comment:
add a constant for 127.0.0.1, so it's easier to refactor in future

Reply:
do we have other place use this ?

Reply:
is_pfs_service_healthy is also used in **check_pfs_service_status**, which will be used to check pfs status after start pfs background

Reply:
update all pfs localhost to ip constant